packages/golden_toolkit/example/lib/src/weather_widgets.dart

```diff
@@ -7,6 +7,9 @@
 /// ***************************************************
 // ignore_for_file: public_member_api_docs
 
+// ignore_for_file: public_member_api_docs
+// ignore_for_file: prefer_const_constructors
```

Comment:
What is the reason for ignoring const constructors?

Reply:
i cant remember what happened. If i didnt open the workspace or something, but i believe i was getting issues with analysis saying i was missing const keywords in places. Rather than have to fix a widget/class i wasnt touching (one that should have already pased the linting), i added the prefer_const_constructors.

I've opened the workspace and removed now locally and it hasnt had any issues. So i will remove.

Interesting point i found when gathering coverage in a work project, in a test file if you construct the object your testing with a const prefix then coverage wasnt taking the fact that the constructor was covered. Removing the const keyword made that work.

Havent retested that in latest flutter stable tbh.

Reply:
I have seen the coverage issue with const constructors, I believe we have some tooling that ignores those.